### DISCUSSION TITLE
Weird q-table problem

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#9391] [General - Components / Directives / etc]
I've spent many hours trying to solve this with no luck. Any insights will be much appreciated...

I have two large components each of whose main element is a q-table. Each one works great by itself, but if they are both open (in separate splitter panes), I get an error saying:
[Vue warn]: You may have an infinite update loop in a component render function.
found in
---> <QTable>
Even if I just resize the window, I get more of these.

Both tables use data from the store, but I've made sure their data is deep cloned.
If I set either of their data to just an empty array, there is no error. But even if I populate their data with dummy values instead of anything from the store, I still get the error.
If I change one component and save, the error comes from the other one and vice versa.

Please help!

**Comment by metalsadman** [comment]
@CrawfordW make a reproduction codepen.

**Reply by CrawfordW** [reply]
It's a very large project - could I just put non-running code there?
Or would it be better to try and make a stripped-down version that still shows the problem (if possible)?

**Comment by CrawfordW** [comment]
Found the problem by process of elimination!
In the unlikely event that someone else has this situation, here's the explanation...
In my app, pages with tables can live in tabs in splitters. A tab can be dragged to a different split pane, so I have to save all the state in the store so it can be re-built when mounted.
In my custom sort method, I was saving the new sort parameters BEFORE doing the sort. I don't understand why Vue thought this could result in an infinite loop, but the problem went away when I waited until after the sort and used a watcher to save the new state.
